distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -75,6 +75,8 @@ export DDF_ON_ERROR="bin/ddf_on_error.sh script "
 # Defines the special on-error Java options
 JAVA_ERROR_OPTS="-XX:OnOutOfMemoryError=\${DDF_ON_ERROR}%p -XX:OnError=\${DDF_ON_ERROR}%p"
 
+export KARAF_SYSTEM_OPTS="-Dpolicy.provider=net.sourceforge.prograde.policy.ProGradePolicy -Djava.security.manager=net.sourceforge.prograde.sm.ProGradeJSM -Djava.security.policy==${DDF_HOME}etc/all.policy -DproGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity"
```

Comment:
Fyi, we had moved the definition of pretty much all java properties into the `system.properties` file a while back. I had done some quick testing back then to make sure that the security manager properties would still be picked up and used correctly that way. Also things like `ddf.home` were set in there in order to avoid some things that were happening when the karaf scripts reference all these variables. Might be worth a quick try since it doesn't require platform specific computation of paths and such.

Reply:
Was this re-tested?